numpy/_core/fromnumeric.py

```diff
@@ -3588,24 +3588,42 @@ def std(a, axis=None, dtype=None, out=None, ddof=0, keepdims=np._NoValue, *,
 
     Notes
     -----
-    The standard deviation is the square root of the average of the squared
-    deviations from the mean, i.e., ``std = sqrt(mean(x))``, where
-    ``x = abs(a - a.mean())**2``.
-
-    The average squared deviation is typically calculated as ``x.sum() / N``,
-    where ``N = len(x)``. If, however, `ddof` is specified, the divisor
-    ``N - ddof`` is used instead. In standard statistical practice, ``ddof=1``
-    provides an unbiased estimator of the variance of the infinite population.
-    ``ddof=0`` provides a maximum likelihood estimate of the variance for
-    normally distributed variables. The standard deviation computed in this
-    function is the square root of the estimated variance, so even with
-    ``ddof=1``, it will not be an unbiased estimate of the standard deviation
-    per se.
+    The definition of the standard deviation of an array is not unique.
+    Assuming the input `a` is a one dimensional NumPy array and `mean` is
+    either provided as an argument or computed as ``a.mean()``, NumPy
+    computes the standard deviation of an array as::
+
+        N = len(a)
+        d2 = abs(a - mean)**2
+        var = d2.sum() / (N - ddof)  # note use of `ddof`
+        std = var**0.5
+
+    Different values of the argument `ddof` are useful in different
+    contexts. NumPy's default ``ddof=0`` corresponds with the expression:
+
+    .. math::
+
+        \sqrt{\frac{\sum_i{|a_i - \bar{a}|^2 }}{N}}
+
+    and is sometimes (for better or for worse) called "population
+    standard deviation" or "uncorrected sample standard deviation".
+
+    Many other libraries define the standard deviation of an array
+    differently, e.g.:
+
+    .. math::
+
+        \sqrt{\frac{\sum_i{|a_i - \bar{a}|^2 }}{N - 1}}
+
+    The use of :math:`N-1` in the denominator is often called "Bessel's
+    correction", and the resulting quantity is sometimed called the "sample
```

Comment:
Similarly here - I really do think you need some justification for the terminology - "sample" in the sense that we consider `a` to be sample from a population, and the variance calculation is biased (to be too low) as an estimate for the population, without the correction.

Reply:
I removed the educational comments. If I add back justification for "sample" and "population", I should also explain the other terms.